mmv1/third_party/terraform/services/compute/resource_compute_instance.go.erb

```diff
@@ -45,6 +45,7 @@ var (
 		"boot_disk.0.initialize_params.0.type",
 		"boot_disk.0.initialize_params.0.image",
 		"boot_disk.0.initialize_params.0.labels",
+        	"boot_disk.0.initialize_params.0.resource_manager_tags",
```

Comment:
Sorry- only last spacing issue, this now has mixed tabs and spaces but we want just tabs